ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly-4.18-upgrade-from-stable-4.18.yaml

```diff
@@ -757,6 +757,8 @@ tests:
     cluster_profile: gcp-qe
     env:
       COMPUTE_NODE_TYPE: t2a-standard-4
+      COMPUTE_SERVICE_ACCOUNT: jiwei-worker-sa@openshift-qe.iam.gserviceaccount.com
```

Comment:
Suggest to apply the change to the following gcp-ipi-xpn-mini-perm-byo-hosted-zone-arm-f60 job too.

Reply:
Same reason as the [above](https://github.com/openshift/release/pull/55925#discussion_r1774720935), thanks!